Packages/com.unity.multiplayer.samples.coop/Utilities/AdditiveSceneLoader.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Unity.Netcode;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+/// <summary>
+/// This NetworkBehavior, when added to a gameobject containing a collider (or multiple colliders) with the IsTrigger
+/// property On, loads a scene additively when there is at least one gameobject with the "Player" tag that enters its
+/// collider, and unloads it when all players leave the collider, after a specified cooldown to prevent it from
+/// repeatedly loading and unloading the same scene.
+/// </summary>
+public class AdditiveSceneLoader : NetworkBehaviour
```

Comment:
To follow the conventions we've already established inside Boss Room, let's add the "Server" prefix to this class/file name.